package-lock.json

```diff
@@ -8806,4 +8806,4 @@
       "dev": true
     }
   }
-}
+}
```

Comment:
Looks like all changes were removed but still have this removing the new line from the end of file.